doc/source/whatsnew/v0.17.0.rst

```diff
@@ -354,15 +362,15 @@ Some East Asian countries use Unicode characters its width is corresponding to 2
 
 .. ipython:: python
 
-   df = pd.DataFrame({u'国籍': ['UK', u'日本'], u'名前': ['Alice', u'しのぶ']})
-   df;
```

Comment:
@dsaxton I guess the semi-colon should stay here? Might be a case of manually changing something that was changed by `black`

Reply:
I think so, it looks like it's being added to suppress output (there are some other cases where this is done to prevent matplotlib from printing text to the screen)

Reply:
Should I add the semi-colon then? I'll let you know if I find any more like this too. @MarcoGorelli @dsaxton

Reply:
yes, please do

Reply:
done!